kernel_gateway/_version.py

```diff
@@ -5,7 +5,7 @@
 
 version_info = (
     2,
-    4,
+    5,
```

Comment:
Shouldn't this version remain on 2.4.0 until the release has been tagged?

Reply:
Never mind... the 'dev' below needs to be removed for the release build anyway, so the version number can be fixed on the same occasion.